apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SpannerClientPartial.cs

```diff
@@ -12,14 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-using Google.Api.Gax;
 using Google.Api.Gax.Grpc;
 using Google.Cloud.Spanner.Common.V1;
-using System;
 
 namespace Google.Cloud.Spanner.V1
 {
-    // Partial class to set up resource-based routing.
+    public partial class SpannerClient
+    {
+        /// <summary>
```

Comment:
This comment is weird, given that it's not a CallSettings, but a SpannerSettings. Not sure whether it *should* be a CallSettings, or a SpannerSettings with better docs. (I realize this commit is just moving it.)